packages/docmanager/src/dialogs.ts

```diff
@@ -220,11 +220,11 @@ namespace Private {
   export function createOpenNode(): HTMLElement {
     let body = document.createElement('div');
     let existingLabel = document.createElement('label');
-    existingLabel.textContent = 'File Path:';
+    existingLabel.textContent = 'Enter Path:';
 
     let input = document.createElement('input');
     input.value = '';
-    input.placeholder = '/path/to/file';
+    input.placeholder = '/path/from/launch/directory';
```

Comment:
Working the word `'relative'` in here somewhere would make it clearer. Maybe `'/path/relative/to/jlab/root'`.